### DISCUSSION TITLE
Docs not in repo?

### DISCUSSION THREAD

**Discussion by filipef101** [gbumps/react-native-screenguard#73] [Q&A] [answered]
Could be worth moving the docs into the repo so community can help mantain. ie was looking to change examples to use aligment from `Aligment` enum/constant instead of the number

**Comment by gbumps** [comment]
Forgot to push, will updated it shortly. Thanks for suggesting!

**Comment by gbumps** [comment, marked as answer]
hi @filipef101, I've updated web docs repo at [here](https://github.com/gbumps/rn-screenguard-docs).

Thanks!
